src/python/pants/backend/codegen/avro/tailor.py

```diff
@@ -49,7 +49,7 @@ async def find_putative_targets(
         PutativeTarget.for_target_type(
             AvroSourcesGeneratorTarget,
             path=dirname,
-            name=None,
+            name="avro",
```

Comment:
I don't know what makes sense for codegen...We could use `avro_sources(name="lib")`, if we think it's unlikely that codegen will be in the same directory as non-codegen sources.

(Reminder: this is temporary)